MonMaraicher/DetailFarmerView.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  DetailFarmerView.swift
+//  MonMaraicher
+//
+//  Created by Yves Charpentier on 27/02/2024.
+//
+
+import SwiftUI
+import MapKit
+
+struct DetailFarmerView: View {
+    
+    @State var selectedFarmer: FarmerPlace?
```

Comment:
The naming here could be improve. The term "selected" has no value here right? DetailsFarmerView doesn't care if that farmer has been selected or not right? The only goal for that view is to display farm details.
SelectedFarmer is the history and that view doesn't care about it.